src/main/java/org/jenkinci/plugins/mock_slave/MockCloud.java

```diff
@@ -63,12 +66,26 @@
     public final Node.Mode mode;
     public final int numExecutors;
     public final String labelString;
+    /** only null when deserializing old configs */
+    private @Nullable Boolean oneShot;
```

Comment:
Check the Javadoc for the annotation. In this case it means that `readResolve` needs to consider the possibility that it is null, but nothing else does.